src/components/molecules/TodoItem.vue

```diff
@@ -0,0 +1,46 @@
+<script setup>
+import AppCheckboxInput from '@/components/atoms/AppCheckboxInput.vue'
+import AppButton from '@/components/atoms/AppButton.vue'
+
+const props = defineProps({
+  name: { type: String, required: true },
+  completed: { type: Boolean, default: false },
+})
+</script>
+
+<template>
+  <div :class="$style['todo-item']">
+    <AppCheckboxInput :checked="props.completed" />
+    <div>
+      <span :class="$style[props.completed ? 'todo-item-completed' : '']">{{ props.name }}</span>
+      <AppButton :class="[$style['button'], $style['item__button--delete']]" label="X" />
+    </div>
+  </div>
+</template>
+
+<style module lang="scss">
```

Comment:
As a suggestion, you can similarly limit the scope of styles within a single component using 'scoped' attribute for the style tag - https://vuejs.org/api/sfc-css-features.html  . In this case, you will be able to define classes simply as strings.